lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/site_parameters.jl

```diff
@@ -0,0 +1,98 @@
+export site_parameters,
+    site_params, hetero_resp, soil, vangenuchten, κ, plant_hydraulics
+
+"""
+    site_parameters(;
+        params,
+        )
+
+Setup parameters for a site
+"""
+function site_parameters(site_ID)
+    default_params_dict = Dict(
+        "US-MOz" => ozark_default_params(),
+        #"US-Ha1" => harvard_default_params(), # need to add those once I create new harvard_default_params() etc
+        #"US-NR1" => niwotridge_default_params(),
+        #"US-Var" => vairaranch_default_params(),
+    )
+
+    if haskey(default_params_dict, site_ID)
+        return default_params_dict[site_ID]
+    else
+        println("This site_ID does not exist")
+    end
+end
+
+# The structs below are necessary and different from ClimaLand/src struct 
+# because of, for example, shared parameters (e.g., porosity in soil or soilco2)
+# or, another example, parameters calculated from other parameters (e.g., κ_solid)
+# the structs below only contain unique params, that are given as number (not computed FTwhere)
+
+struct hetero_resp # differs from SoilCO2ModelParameters because of porosity
+    θ_a100::FT
+    D_ref::FT
+    b::FT
+    D_liq::FT
+    α_sx::FT
+    Ea_sx::FT
+    kM_sx::FT
+    kM_o2::FT
+    O2_a::FT
+    D_oa::FT
+    p_sx::FT
+end
+
+struct soil # Differs from EnergyHydrologyParameters because k_dry etc are computed. Will be fixed in PR SpaceVaryingInput #485
+    ν::FT
+    ν_ss_quartz::FT
+    ν_ss_om::FT
+    K_sat::FT
+    S_s::FT
+    θ_r::FT
+    ν_ss_gravel::FT
+    z_0m::FT
+    z_0b::FT
+    emissivity::FT
+    PAR_albedo::FT
+    NIR_albedo::FT
+end
+
+struct vangenuchten # I don't think these have a struct in src
+    vg_n::FT
+    vg_α::FT
+end
+
+struct κ # same, these are seen in soil_heat_parameterizations.jl, but in no struct. May be fixed in PR SpaceVaryingInput #485
```

Comment:
I think a more descriptive name for the struct would be nice (what does kappa represent?)